src/title/TitleStream.js

```diff
@@ -68,37 +68,37 @@ const TitleStream = (props: Props) => {
   const _ = useContext(TranslationContext);
 
   return (
-    <View style={componentStyles.outer}>
-      <View style={componentStyles.streamRow}>
-        <StreamIcon
-          style={styles.halfMarginRight}
-          isMuted={!stream.in_home_view}
-          isPrivate={stream.invite_only}
-          color={color}
-          size={20}
-        />
-        <Text style={[styles.navTitle, { color }]} numberOfLines={1} ellipsizeMode="tail">
-          {stream.name}
-        </Text>
-      </View>
-      {isTopicNarrow(narrow) && (
-        <TouchableWithoutFeedback
-          onLongPress={() => {
-            showHeaderActionSheet({
-              showActionSheetWithOptions,
-              callbacks: { dispatch, _ },
-              backgroundData,
-              stream: stream.name,
-              topic: topicOfNarrow(narrow),
-            });
-          }}
-        >
+    <TouchableWithoutFeedback
+      onLongPress={() => {
+        showHeaderActionSheet({
+          showActionSheetWithOptions,
+          callbacks: { dispatch, _ },
+          backgroundData,
+          stream: stream.name,
+          topic: topicOfNarrow(narrow),
```

Comment:
That `topicOfNarrow(narrow)` should go behind an `isTopicNarrow(narrow)` check; else there's an error when long-pressing the header if it's a stream narrow.

Reply:
Done, thanks for catching that!